File: frontend/src/app/model/carOwner.ts
import { Car } from './car';
import { Order } from './order';

export interface CarOwner {
  id: number;
  name: string;
  lastName: string;
  phoneNumber: string;
  carsId: number[];
  cars: Car[];
  carIds: Car;
  orders: Order[];
  ordersId: number[];
}

File: api/src/main/java/com/example/autoservice/model/CarOwner.java
package com.example.autoservice.model;

import lombok.*;
import javax.persistence.*;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(onlyExplicitlyIncluded = true)
@Entity
@Table(name = "owners")
public class CarOwner {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @EqualsAndHashCode.Include
    private Long id;
    private String name;
    private String lastName;
    private String phoneNumber;
//    @OneToMany
//    private List<Car> cars;
//    @OneToMany
//    private List<Order> orders;
}
